build/commands/lib/config.js

```diff
@@ -179,7 +179,6 @@ Config.prototype.buildArgs = function () {
     is_asan: this.isAsan(),
     enable_full_stack_frames_for_profiling: this.isAsan(),
     v8_enable_verify_heap: this.isAsan(),
-    fieldtrial_testing_like_official_build: true,
```

Comment:
can you elaborate on why is this needed?

Reply:
the variations service is not loading the seed w/o passing the URL via cli args, so was looking through old PRs that are disabling variations/trials and might fix this. Found the above and was waiting on @bridiver or @mkarolin to review

Reply:
cc @jumde

Reply:
@diracdeltas just to clarify we wouldn't load any chromium trials - we host our own seed file under variations.brave.com as per https://github.com/brave/security/issues/241

Reply:
@moritzhaller - We should not enable field-trials. It increases the fingerprinting surface for the browser significantly.

Reply:
Looks like the fingerprinting risk has been discussed in https://github.com/brave/security/issues/241 - In that case, we should be good. We should make sure that chromium trials are not enabled with this change. @fmarier @diracdeltas - let me know if that works.

Reply:
this change doesn't look correct to me, we should not be setting `FIELDTRIAL_TESTING_ENABLED` and that's what this change does https://github.com/brave/brave-core/pull/6940/files#diff-f2c13b8af4947e7149b6a992245b7e3d202d18c282ed8fd851e91caee213f9aaR17